README.md

```diff
@@ -5,223 +5,43 @@
-  --genesis-path <CONFIG>/genesis.toml
-```
-
-which will generate `miden-node.toml` and `genesis.toml` files. The latter controls the accounts that the genesis block
-will be spawned with and by default includes a basic fungible faucet account. You can modify this file to add/remove
-accounts as desired.
-
-Next, bootstrap the chain by generating the genesis data:
-
-```sh
-miden-node make-genesis \
-  --inputs-path  <CONFIG>/genesis.toml \
-  --output-path <STORAGE>/genesis.dat
-```
-
-which will create `genesis.dat` and an `accounts` directory containing account data based on the `genesis.toml` file.
-
-Modify the `miden-node.toml` configuration file such that the `[store]` paths point to our `<STORAGE>` folder:
-
-```toml
-[store]
-database_filepath = "<STORAGE>/miden-store.sqlite3"
-genesis_filepath  = "<STORAGE>/genesis.dat"
-blockstore_dir    = "<STORAGE>/blocks"
-```
-
-Finally, configure the node's endpoints to your liking.
-
-### Systemd
-
-An example service file is provided [here](packaging/miden-node.service). If you used the Debian package installer then
-this service was already installed alongside it.
-
-### Running the node
-
-Using the node configuration file created in the previous step, start the node:
-
-```sh
-miden-node start \
-  --config <CONFIG>/miden-node.toml \
-  node
-```
-
-or alternatively start the systemd service if that's how you wish to operate:
-
-```sh
-systemctl start miden-node.service
-```
-
-The node will use local provers for blocks and batches, but you can configure the node to use remote provers by adding the following to the `miden-node.toml` file in the `[block-producer]` section:
-
-```toml
-batch_prover_url = "<BATCH_PROVER_URL>"
-block_prover_url = "<BLOCK_PROVER_URL>"
-```
-
-### Monitoring and telemetry
-
-Please see our operator documentation [here](docs/operator.md).
-
-## Updating
-
-We currently make no guarantees about backwards compatibility. Updating the node software therefore consists of wiping
-all existing data and re-installing the node's software again. This includes regenerating the configuration files and
-genesis block as these formats may have changed. This effectively means every update is a complete reset of the
-blockchain.
-
-First stop the currently running node or systemd service then remove all existing data. If you followed the
-[Setup](#setup) section, then this can be achieved by deleting all information in `<STORAGE>`:
-
-```sh
-rm -rf <STORAGE>
-```
+Documentation, tutorials and guides for the current Miden version (aka testnet) can be found
+[here](https://0xpolygonmiden.github.io/miden-docs/), including an operator manual and gRPC reference guide. This is
+your one-stop-shop for all things Miden.
 
 > [!WARNING]
-> Failure to remove existing node data could result in strange behaviour.
+> For node operators living on the development edge, we also host the latest documentation
+> [here](https://0xpolygonmiden.github.io/miden-node/index.html).
```

Comment:
nit: I'm not sure this should be in a "warning" callout. If we wanted to call more attention to it, maybe "note" would be better? But also, regular text is probably fine.